tests/scripts/extract_alerts.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+INPUT_YAML="$1"
+OUTPUT_DIR="$2"
+
+mkdir -p "$OUTPUT_DIR"
+
+# Define RULES_FILES and output files
+RULES_FILES=(
+  "rhods-dashboard-alerting.rules"
+  "model-mesh-alerting.rules"
+  "trustyai-alerting.rules"
+  "odh-model-controller-alerting.rules"
+  "workbenches-alerting.rules"
+  "data-science-pipelines-operator-alerting.rules"
+  "kserve-alerting.rules"
+  "kueue-alerting.rules"
+  "ray-alerting.rules"
+  "codeflare-alerting.rules"
+  "trainingoperator-alerting.rules"
+)
+
+ALERT_FILES=(
+  "dashboard_alerts.yaml"
+  "model_mesh_alerts.yaml"
+  "trustyai_alerts.yaml"
+  "model_controller_alerts.yaml"
+  "workbenches_alerts.yaml"
+  "data_science_pipelines_operator_alerts.yaml"
+  "kserve_alerts.yaml"
+  "kueue_alerts.yaml"
+  "kuberay_alerts.yaml"
+  "codeflare_alerts.yaml"
+  "training_operator_alerts.yaml"
+)
+
+for i in "${!RULES_FILES[@]}"; do
+  RULE_FILE="${RULES_FILES[$i]}"
+  ALERT_FILE="${ALERT_FILES[$i]}"
```

Comment:
Any reason to have RULES_FILES and ALERT_FILES as separated arrays in sync by index instead of just an associative array?

Reply:
> So I tried with an associative array but I wasn't able to successfully loop over the arrays. I dig in more if you think it's a better idea. (It's definitely a cleaner code)

```
% cat 1.bash
#!/bin/bash

declare -A ARR=([key1]=value1 [key2]=value2)

for key in "${!ARR[@]}"; do
    echo "key: $key"
    echo "value: ${ARR[$key]}"
done

% bash 1.bash
key: key2
value: value2
key: key1
value: value1
```
?

Reply:
So, I was trying the above format. But the problem was running the bash version in my system. It needed a `#!/usr/bin/env bash`